### ISSUE TITLE
Multiple Errors in Checks.py [Windows]

### ISSUE THREAD

**Issue by apastel** [gilesknap/gphotos-sync#145]
The update to 2.10.2 is breaking for me when the Checks.py module is trying to get the max path length of my Windows system.

It looks like a Unix subprocess that does the check, so it's expected that this will fail on Windows (hence the try/except block) but the print() statement in the except block is itself raising an exception.

```
C:\>pipenv run gphotos-sync --album "Concerts" "C:\New folder"
Traceback (most recent call last):
  File "c:\users\alexr\.virtualenvs\-snfpu_4o\lib\site-packages\gphotos\Checks.py", line 52, in get_max_path_length
    ['getconf', 'PATH_MAX', '/'])
  File "c:\program files\python37\Lib\subprocess.py", line 395, in check_output
    **kwargs).stdout
  File "c:\program files\python37\Lib\subprocess.py", line 472, in run
    with Popen(*popenargs, **kwargs) as process:
  File "c:\program files\python37\Lib\subprocess.py", line 775, in __init__
    restore_signals, start_new_session)
  File "c:\program files\python37\Lib\subprocess.py", line 1178, in _execute_child
    startupinfo)
FileNotFoundError: [WinError 2] The system cannot find the file specified

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\program files\python37\Lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\program files\python37\Lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Users\alexr\.virtualenvs\-SNfpu_4o\Scripts\gphotos-sync.exe\__main__.py", line 7, in <module>
  File "c:\users\alexr\.virtualenvs\-snfpu_4o\lib\site-packages\gphotos\Main.py", line 421, in main
    GooglePhotosSyncMain().main()
  File "c:\users\alexr\.virtualenvs\-snfpu_4o\lib\site-packages\gphotos\Main.py", line 389, in main
    args = self.fs_checks(root_folder, args)
  File "c:\users\alexr\.virtualenvs\-snfpu_4o\lib\site-packages\gphotos\Main.py", line 365, in fs_checks
    Checks.get_max_path_length(root_folder)
  File "c:\users\alexr\.virtualenvs\-snfpu_4o\lib\site-packages\gphotos\Checks.py", line 55, in get_max_path_length
    print('calling getconf failed - error:', traceback=True)
TypeError: 'traceback' is an invalid keyword argument for print()
```

I tried googling for `'traceback' is an invalid keyword argument for print()` but I couldn't find any examples of `traceback` actually being used as an argument for print(). So I tried removing the `traceback=True` argument from that line and from the print statement in `get_max_filename_length` and then instead got an error `name deprint not defined`. It says `deprint` instead of `print` on line 68. Very possibly a typo. 

I realize I could make a PR to fix both but I want to confirm first if this is a bug; I assume there are other Windows users who are getting by on the new release somehow.

**Comment by gilesknap** [comment]
Thanks for the report. I will look into this.

I'm not sure how this for past the CI tests since they are executed on
Windows, Mac and Linux. But there should also not be any print statements
in the code!

**Comment by gilesknap** [comment]
Yep checks.py is broken for Windows. And it won't break in my CI Travis tests.

This must be something to do with the Travis VM having access to Linux APIs??

Anyway, I'll get it fixed soon. Might have to crack out my dual boot to windows for testing!

**Comment by gilesknap** [comment]
This should be fixed in 2.10.3
Please re-open if you still have issues.
